ulid/__init__.py

```diff
@@ -22,13 +22,17 @@
 __version__ = version("python-ulid")
 
 
-class validate_type:  # noqa: N801
-    def __init__(self, *types: Any) -> None:
+T = TypeVar("T", bound=type)
```

Comment:
nit: Isn't somehow everything a sublcass of `type`? I mean, does the `bound`-parameter change anything?

Reply:
Needed `bound=type` to indicate it was a "class" for the 2nd argument of the later `isinstance()`